### ISSUE TITLE
Pressing Ctrl+W in thunderbird...

### ISSUE THREAD

**Issue by o-pikozh** [foudfou/FireTray#184]
... causes thunderbird to quit without leaving icon in sys tray

**Comment by foudfou** [comment]
...for the last tab, and as Ctrl-Q does, that's right. Obviously we needs a way of quitting the application.
To hide the application, you need to send a *close* event to the window (Alt-F4 on Windows for ex.) You can also create a keyboard shortcut in your environment and bind it to `thunderbird -firetrayShowHide`.
Does this help ?

**Comment by o-pikozh** [comment]
No, I think that closing last tab should be caught and overridden with hide-to-tray too (at least, when checkbox is set in preferences; i.e. 3rd checkbox near the "closing hides-to-tray" and "minimizing hides-to-tray").
Although, I certainly agree that user needs a way for quitting application, it should be quite explicit way (for tray-staying application), not the one that is typically used to close non-tray-bound application (just imagine if Skype quits on Ctrl+W).
But I understand that it may be technically hard/impossible.

**Comment by foudfou** [comment]
OK. I'll leave this one open so other users can comment.
Although I must say, I won't be able to work on it soon.

**Comment by Zugunruhekami** [comment]
Hello thanks for your nice add-on! But I would strongly suggest a feature requested here.

The reason being, if I have multiple mails opened in several tabs, and I read through all of them and then press ctrl+W to close them until having only the inbox, I would do that again to just close the last tab and go on doing my other work, leaving Thunderbird running in the background in a system tray.

What you suggested, by pressing Alt-F4 in the last tab to achieve this effect, would break the flow of keeping pressing ctrl+W. Thus I strongly suggest you can come up this feature in the future, or at least give us an option for this.

I would think most users coming from OS X would strongly support this one.

**Comment by Zugunruhekami** [comment]
The other reason being, ctrl+W would behave exactly as ctrl+Q if not implementing otherwise, which doesn't make sense, since the former is about closing the window, the latter being about closing the app. Now they are mixed together.

I strongly suggest we do as another user pointed out.
"Expected: Ctrl+W should be the same like X close of the window or Alt+F4."
In this way we can separate the two usages by using your add-on.

**Comment by Greyvend** [comment]
Any news about this issue? I found today this unexpected behavior. All other tray based applications I use (for example, Steam, Clementine, Skype) don't completely exit but minimize to tray on `Ctrl` + `W`. It would be nice if FireTray followed this route.

**Comment by Markieta** [comment]
+1 this but it seems the project is dead.
